src/main/java/net/imglib2/algorithm/morphology/StructuringElements.java

```diff
@@ -88,6 +88,7 @@ public class StructuringElements
 	 * This methods relies on heuristics to determine automatically what
 	 * decomposition level to use.
 	 *
+	 * @implNote op name='create.disk', type=Function
```

Comment:
Should these be Ops?

Reply:
Also: should this `create` Op also be an `engine.create` alias? If not: does it really belong in the `create` namespace?"

Reply:
While I like the idea of having `create` Ops for `Shapes`, I again don't think that these methods are quite general enough, so I'm just not going to opify them, *yet*.